libsolidity/formal/SymbolicState.cpp

```diff
@@ -96,6 +96,11 @@ smtutil::Expression SymbolicState::balance(smtutil::Expression _address) const
 	return smtutil::Expression::select(balances(), std::move(_address));
 }
 
+smtutil::Expression SymbolicState::blobhash(smtutil::Expression _blockNumber) const
+{
```

Comment:
```suggestion
smtutil::Expression SymbolicState::blobhash(smtutil::Expression _blobIndex) const
{
```
Note that for `blobhash` the argument is not a block number, but the index of the blob commitment within the transaction. See: https://github.com/ethereum/solidity/blob/a4179558040a113674e4fab4d3728bb8e13b0ee5/test/tools/yulInterpreter/EVMInstructionInterpreter.cpp#L701

Reply:
Thanks, I missed this when I was copy-pasting from `blockhash` :)